codersdk/workspaces.go

```diff
@@ -253,6 +253,9 @@ func (c *Client) UpdateWorkspace(ctx context.Context, id uuid.UUID, req UpdateWo
 
 // UpdateWorkspaceAutostartRequest is a request to update a workspace's autostart schedule.
 type UpdateWorkspaceAutostartRequest struct {
+	// Schedule is expected to be of the form `CRON_TZ=<IANA Timezone> <min> <hour> * * <dow>`
```

Comment:
Is there a reason we don't deprecate this field and instead make `CronTimezone` the property on the struct?

Reply:
Could definitely do this! Main reason was to document. Will file a follow-up.